Multisig/Features/Security/SensitiveStore.swift

```diff
@@ -0,0 +1,178 @@
+//
+//  SensitiveStore.swift
+//  Multisig
+//
+//  Created by Dirk Jäckel on 30.11.22.
+//  Copyright © 2022 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+class SensitiveStore: EncryptedStore {
+    var initialized = false
+    private let store: KeychainItemStore
+    static let sensitivePublicKeyTag = "global.safe.sensitivePublicKeyTag"
+    static let sensitivePublicKEKTag = "global.safe.sensitivePublicKEKTag"
+    static let sensitivePrivateKEKTag = "global.safe.sensitivePrivateKEKTag"
+    static let sensitiveEncryptedPrivateKeyTag = "global.safe.sensitive.private.key.as.encrypted.data"
+    static let derivedPasswordTag = "global.safe.password.as.data"
+
+    init(_ store: KeychainItemStore) {
+        self.store = store
+    }
+
+    func isInitialized() -> Bool {
```

Comment:
instead of this, you could declare the `initialized` the following way:

```
private (set) var isInitialized: Bool
```

This way, you can set it only inside the class, but it is available to read for outside.

Reply:
do we need this variable, at all? if the `initialize()` fails, it will throw error anyway

Reply:
This var does not work for what I wanted to do. It was supposed to check whether the Sensitive Store has been initialized. Across app starts. To do that we would need to check whether there is, for example, a sensitive key already in the key chain.